autoware_cmake/cmake/autoware_package.cmake

```diff
@@ -48,6 +48,11 @@ macro(autoware_package)
   find_package(ament_cmake_auto REQUIRED)
   ament_auto_find_build_dependencies()
 
+  # Set common system includes
+  include_directories(SYSTEM
+    ${EIGEN3_INCLUDE_DIR}
+  )
+
```

Comment:
@esteve let's add this and create we can open up a PR to remove it by adding it to necessary packages later on, what do you think?

Reply:
@xmfcx sounds good to me